docs/decisions/app/0005-server-rendering.md

```diff
@@ -0,0 +1,73 @@
+## Context and Problem Statement
+
+Next.js provides multiple ways of rendering a website, which have different commands: `next start` vs `next export`. For the platform, we need to determine which should be the default.
+
+## Considered options
+
+1.  Server rendering (`next start`): Generates the full HTML markup for a page on the server in response to navigation. Minimal client-side JS is possible, even for personalized sites.
+2.  Prerendering (`next export`): At compile time, a separate HTML file is generated for each URL. Only the initial state of the page is generated as static HTML. To display personalized data, client-side JS is required and the page's DOM is updated.
+
+## Decision drivers
+
+1.  The selected option should represent what we consider the preferred approach for the types of websites we're typically building with Next.js: authenticated, personalized web applications (claimant portals, compare tools, case management systems, etc).
+2.  Reduce the need for third party dependencies or custom code, when a native option works just as well.
+3.  A (reasonable) increase in cloud costs is acceptable if it results in a system that is more maintainable for software teams and the government in the long term.
+4.  Developer experience should not come at the expense of end user experience.
+
+## Decision Outcome
+
+Server rendering is the best option when the web application requires "live" data, such as the personalized sites we often build at Nava, like claimant portals or case management systems. Server rendering requires more upfront effort on the infra side, but it enables teams to achieve a clearer separation of concerns, and write less application code in the long run. This can translate to web applications that work well for a large spectrum of device and network conditions.
+
+If a project team is building a site that renders the same content for every user, they can change their application to utilize [Next.js's static HTML export functionality](https://nextjs.org/docs/advanced-features/static-html-export) (prerendering). This flexibility to do either server rendering or prerendering within the same React framework is one benefit to using Next.js.
+
+## Pros and Cons of the Options
+
+### Server rendering
+
+Pros
```

Comment:
Can you do dynamic routing? Or are we still stuck with file-based routing?

Reply:
I think you can technically do dynamic routes in both options, however it's easier in server rendering. I have an item below that I think touches on this:

> Low effort to implement pretty [dynamic routes](https://nextjs.org/docs/routing/dynamic-routes) (e.g `/claim/:claim_id`)